Product/Production/Services/DocumentDataSubmissionCore/RegisterDocumentSet-bRequest.xml

```diff
@@ -0,0 +1,208 @@
+                        </rim:ValueList>
+                    </rim:Slot>
+                    <rim:Name>
+                        <rim:LocalizedString value="History and Physical"/>
+                    </rim:Name>
+                </rim:Classification>
+                <rim:Classification id="cl03" classificationScheme="urn:uuid:f4f85eac-e6cb-4883-b524-f2705394840f" classifiedObject="Document01" nodeRepresentation="1.3.6.1.4.1.21367.2006.7.101">
+                    <rim:Slot name="codingScheme">
+                        <rim:ValueList>
+                            <rim:Value>Connect-a-thon confidentialityCodes</rim:Value>
+                        </rim:ValueList>
+                    </rim:Slot>
+                    <rim:Name>
+                        <rim:LocalizedString value="Clinical-Staff"/>
+                    </rim:Name>
+                </rim:Classification>
+                <rim:Classification id="cl04" classificationScheme="urn:uuid:a09d5840-386c-46f2-b5ad-9c3699a4309d" classifiedObject="Document01" nodeRepresentation="CDAR2/IHE 1.0">
+                    <rim:Slot name="codingScheme">
+                        <rim:ValueList>
+                            <rim:Value>Connect-a-thon formatCodes</rim:Value>
+                        </rim:ValueList>
+                    </rim:Slot>
+                    <rim:Name>
+                        <rim:LocalizedString value="CDAR2/IHE 1.0"/>
+                    </rim:Name>
+                </rim:Classification>
+                <rim:Classification id="cl05" classificationScheme="urn:uuid:f33fb8ac-18af-42cc-ae0e-ed0b0bdb91e1" classifiedObject="Document01" nodeRepresentation="Outpatient">
+                    <rim:Slot name="codingScheme">
+                        <rim:ValueList>
+                            <rim:Value>Connect-a-thon healthcareFacilityTypeCodes</rim:Value>
+                        </rim:ValueList>
+                    </rim:Slot>
+                    <rim:Name>
+                        <rim:LocalizedString value="Outpatient"/>
+                    </rim:Name>
+                </rim:Classification>
+                <rim:Classification id="cl06" classificationScheme="urn:uuid:cccf5598-8b07-4b77-a05e-ae952c785ead" classifiedObject="Document01" nodeRepresentation="General Medicine">
+                    <rim:Slot name="codingScheme">
+                        <rim:ValueList>
+                            <rim:Value>Connect-a-thon practiceSettingCodes</rim:Value>
+                        </rim:ValueList>
+                    </rim:Slot>
+                    <rim:Name>
+                        <rim:LocalizedString value="General Medicine"/>
+                    </rim:Name>
+                </rim:Classification>
+                <rim:Classification id="cl07" classificationScheme="urn:uuid:f0306f51-975f-434e-a61c-c59651d33983" classifiedObject="Document01" nodeRepresentation="34108-1">
+                    <rim:Slot name="codingScheme">
```

Comment:
What is this xml used for?

Reply:
It's used in a class called XDRMessageHelper which as far as I can tell is only used for a test called XDRHelperTest